### DISCUSSION TITLE
Hp ProBook 6460b - is possible use the OC package?

### DISCUSSION THREAD

**Discussion by spn0381** [chris1111/HP-Probook-EliteBook-Package-Creator-OC#179] [General]
Hi Chris! I have a hp probook 6460 with installed Catalina (with your "old guide") and I'm searching to upgrade it with Big Sur or Ventura... Seems that the 6 series isn't supported by OC package, I would like to understand why, there are hw limits or other stuffs?

thank you in advance!

**Comment by chris1111** [comment]
Hi @spn0381  Because I starting a new guide with OC from 7 series here then I add the 8 series

**Reply by spn0381** [reply]
Thank you Chris … do you Thank thats impossible make work the 6 series with ventura !? Thank you

**Comment by chris1111** [comment]
It is possible with OCLP but I suggest to you to use OpenCore

**Reply by spn0381** [reply]
But oclp doesn’t work only with a genuine Apple device ? How I can use it with pro book ?

**Reply by chris1111** [reply]
Yes it work on Hackintosh

**Reply by spn0381** [reply]
I will test in the weekend starting from the actual catalina installation... I'm really surprised that OCPL work also on hackintosh... in the documentation is not reported...

**Comment by chris1111** [comment]
You have to use your EFI (Booting Sonoma), dont use Opencore fron OCLP, use only `RootPatch`
[RootPatch](https://github.com/chris1111/HP-Probook-EliteBook-Package-Creator-OC/blob/master/macOS%20Ventura%20Support.md)

**Reply by spn0381** [reply]
I will try in the next hour … my only doubt is the graphic card… seems not compatible with Ventura or sonoma…

**Comment by chris1111** [comment]
> I will try in the next hour … my only doubt is the graphic card… seems not compatible with Ventura or sonoma…



Patch HD 3000 is included in OCLP

**Reply by spn0381** [reply]
I’m configuring the open core to boot … there is a way to use the old clover:
USB (elsewhere I have to use usbtoolbox)
Dstd

lot of thanks 🙏

**Reply by spn0381** [reply]
Seems too hard for me to create the oc package myself… sandy bridge is very difficult for me to configure…

@chris1111 : it’s a crazy idea buy a quad i7 with intel hd 4000 trying to use it? Or buy an 6470 mainboard using all the other components of 6460…
